Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -52,6 +52,42 @@ class MyTestInParallel_2(EditorParallelTest):
 logger = logging.getLogger(__name__)
 
 
+def _split_batched_editor_log_file(workspace, starting_path, destination_file, log_file_to_split):
+    """
+    Splits a batched editor log file into separate log files for each test case in the log
+    :param path: the path to the logs to split
+    :param log_file_to_split: the log file to split
+    """
+    # text that designates the start of logging for a new test
+    test_case_split = ".py (testcase )"
+    dir_name = os.path.dirname(starting_path)
+
+    # the current log we are writing to
+    current_new_log = None
+    index = 1
+    while not current_new_log:
+        current_new_log_path = os.path.join(dir_name, f"SetUp" + str(index) + ".log")
+        current_new_log = open(current_new_log_path, "a+")
+        index += 1
+
+    # loop through the log to split, and write to the split logs
+    with open(destination_file) as log_file:
+        for line in log_file:
+            split_line = line.split(test_case_split)
+            if len(split_line) > 1:
+                # found a new test case, we need to split the log, line below is an example of what we're splitting
+                # <13:22:34> (python) - Running automated test: C:\\Git\\o3de\\AutomatedTesting\\Gem\\PythonTests\\largeworlds\\dyn_veg\\EditorScripts\\LayerSpawner_InstancesPlantInAllSupportedShapes'
+                new_log_name = split_line[0].split("\\")[-1] + ".log"
```

Comment:
Has this been tested with a Linux log? Their slashes may be different than Windows, but I'm not 100% sure.